src/components/FeatureViews/Scheduler/ScheduleForm.js

```diff
@@ -0,0 +1,130 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as calenderActions from '../../../actions/calenderActions';
+import PropTypes from 'prop-types';
+import React, { Component, useState } from 'react';
+import BackButton from "../../BackButton.js";
+import SessionActions from "./SessionActions";
+import '../../../theme/theme.scss';
+import './scheduler.scss'
+import { NavLink } from "react-router-dom";
+import FullCalendar from '@fullcalendar/react';
+import interactionPlugin from '@fullcalendar/interaction'
+import resourceTimelinePlugin from '@fullcalendar/resource-timeline';
+
+//Material UI Imports
+import Grid from "@material-ui/core/Grid";
+import Paper from "@material-ui/core/Paper";
+
+import { Divider, LinearProgress, Typography, ListItem, RadioGroup } from "@material-ui/core";
```

Comment:
Use the separate import syntax that the other imports from Material UI use - this helps improve loading times by reducing bundle size. Example:

    import Typography from "@material-ui/core/Typography"